.golangci.yml

```diff
@@ -17,12 +17,14 @@ linters:
   - bidichk
   - bodyclose
   - containedctx
+  - decorder
```

Comment:
It looks like most (maybe all?) of the checks in this are disabled by default. We shouldn't add this as is - and I think we'd need a discussion about how to implement it. https://golangci-lint.run/usage/linters/#decorder

When I enable the checks locally there's a huge number of findings. This - `multiple "type" declarations are not allowed; use parentheses instead` especially seems problematic and not idiomatic to me

Reply:
Removed.